.github/ISSUE_TEMPLATE.md

```diff
@@ -2,13 +2,19 @@
 <!-- Amend ? below with the correct information -->
 
 # Homebrew summary
-- VitaDB: <http://vitadb.rinnegatamante.it/#/info/?>
+- VitaDB: <http://vitadb.rinnegatamante.it/#/info/?><!-- Replace "?" with homebrew hash -->
```

Comment:
Shouldnt it be homebrew id instead?, they are a number `https://vitadb.rinnegatamante.it/#/info/1144`